### ISSUE TITLE
BUG: The classmethod is_installed on DebianPackage always fails when package is not installed

### ISSUE THREAD

**Issue by codylane** [philpep/testinfra#321]
I'm a little caught off guard by this and I cannot seem to find any open issues the issue I've noticed.

This snippet will always **FAIL** if the package being queried does not exist.  
https://github.com/philpep/testinfra/blob/master/testinfra/modules/package.py#L87-L91

## How to reproduce
1. Fire up a Ubuntu 16.04 instance
2. As an example, we will first query for a package that we know exists on this 16.04 instance.

`$ cat /etc/os-release`

```
NAME="Ubuntu"
VERSION="16.04.4 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.4 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
```

`$ dpkg-query -f '${Status}\n' -W bash`

```
install ok installed
```

`$ echo $?`
```
0
```

3. Now that we know `bash` exists let's try this same query with a package we know is not installed on the system

`$ dpkg-query -f '${Status}\n' -W a-dummy-name-that-does-not-exist`

```
dpkg-query: no packages found matching a-dummy-name-that-does-not-exist
```

`$ echo $?`
```
1
```

I cannot find any issues about this and it appears to have been a bug for some time or I can only assume that others have not tried to use the `is_installed` method on Ubuntu 16.04 when a package is not installed.  

This was introduced almost **2 YEARS ago** via this commit: https://github.com/philpep/testinfra/commit/a68908c46bffe447f4fb1ef2b1796b6ff1f7e672

The lack of formal testing surrounding this commit is mind boggling for a library dubbed as a `Testing library`.  It's shocking to me to only find  `two` tests for `is_installed`.
https://github.com/philpep/testinfra/blob/1932c810b8697bb41d3ea1890f9e95e28228c803/test/test_modules.py#L32-L68

## Proposed Solution
 I'd love to help be a maintainer/contributor of this project because I see the value it provides and its very simple to use.  I'm a big fan!  However, I will admit, I am NOT a fan of the lack of tests surrounding this library and I find it concerning to see what feels like new features being prioritized over the issues.

1. I'd like to suggest that the solution to the issue be consistent with those in the rest of `package.py` Looking at the others I see the following using `run_test` instead of `check_output` because the later raises an `AssertionError` when `rc != 0`

Here' s a potential fix and I would be happy to submit a pull request. :)

```
class DebianPackage(Package):

    @property
    def is_installed(self):
        return self.run_test("dpkg-query -f '${Status}' -W %s" % (
            self.name)).rc == 0
```

But if I do submit a PR, I feel there should be more tests to make sure this works across all `supported OS` versions whatever that may mean.  I generally only assume a `supported OS` is one that has tests so looking at the current tests I would only be writing tests for...
 ```
    "debian_stretch"
    "ubuntu_xenial"
```
But given my past experience as a contributor to this project, I was informed that I can only introduce one change per pull request, but testing generally requires changes to more than one file.
https://github.com/philpep/testinfra/pull/226#issuecomment-321991140

**Comment by codylane** [comment]
You rock!  Thanks @philpep for the quick review and fix.

**Comment by philpep** [comment]
This will be fixed in 1.13.1 which I plan to release today. Thanks for reporting!

**Comment by codylane** [comment]
Awesome!  You are welcome and I'm always glad to help. I really like this library. :)
